pages/common/boot.md

```diff
@@ -0,0 +1,23 @@
+# boot
+
+> Build tooling for Clojure.
+
+- Generate scaffolding for a new project based on a template:
+
+`boot -d boot/new new --template {{template_name}} --name {{project_name}}`
+
+- Start a REPL session either with the project or standalone:
+
+`boot repl`
+
+- Build for development (if using the boot/new template):
```

Comment:
How would one build if _not_ using the boot/new template ?

Reply:
The "uberjar" as far as I understand is the canonical way. I moved it to above the other ways that are part of the recommended templated ways.